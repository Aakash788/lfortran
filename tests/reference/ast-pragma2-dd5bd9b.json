{
    "basename": "ast-pragma2-dd5bd9b",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/pragma2.f90",
    "infile_hash": "c76c30115a39aa0c98b24173c8517df1fb525bf101023b12dc38ca3e",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-pragma2-dd5bd9b.stdout",
    "stdout_hash": "21ef66a11c770eaf57a58e464714465138c22b24eceb589ce9992502",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}