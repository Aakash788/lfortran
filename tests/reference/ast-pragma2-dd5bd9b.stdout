(TranslationUnit
    [(Program
        pragma2
        (TriviaNode
            [(EndOfLine)
            (Comment
                "!LF$ attributes simd :: A"
            )]
            []
        )
        []
        []
        [(Declaration
            (AttrType
                TypeReal
                []
                ()
                ()
                None
            )
            []
            [(A
            [(1
            8
            DimensionExpr)]
            []
            ()
            None
            ())]
            ()
        )]
        [(=
            0
            A
            1
            ()
        )]
        []
    )]
)
